packages/flutter/lib/src/material/outlined_button.dart

```diff
@@ -0,0 +1,362 @@
+    bool enableFeedback,
+  }) {
+    final MaterialStateProperty<Color> foregroundColor = (onSurface == null && primary == null)
+      ? null
+      : _OutlinedButtonDefaultForeground(primary, onSurface);
+    final MaterialStateProperty<Color> overlayColor = (primary == null)
+      ? null
+      : _OutlinedButtonDefaultOverlay(primary);
+    final MaterialStateProperty<MouseCursor> mouseCursor = (enabledMouseCursor == null && disabledMouseCursor == null)
+      ? null
+      : _OutlinedButtonDefaultMouseCursor(enabledMouseCursor, disabledMouseCursor);
+
+    return ButtonStyle(
+      textStyle: ButtonStyleButton.allOrNull<TextStyle>(textStyle),
+      foregroundColor: foregroundColor,
+      backgroundColor: ButtonStyleButton.allOrNull<Color>(backgroundColor),
+      overlayColor: overlayColor,
+      shadowColor: ButtonStyleButton.allOrNull<Color>(shadowColor),
+      elevation: ButtonStyleButton.allOrNull<double>(elevation),
+      padding: ButtonStyleButton.allOrNull<EdgeInsetsGeometry>(padding),
+      minimumSize: ButtonStyleButton.allOrNull<Size>(minimumSize),
+      side: ButtonStyleButton.allOrNull<BorderSide>(side),
+      shape: ButtonStyleButton.allOrNull<OutlinedBorder>(shape),
+      mouseCursor: mouseCursor,
+      visualDensity: visualDensity,
+      tapTargetSize: tapTargetSize,
+      animationDuration: animationDuration,
+      enableFeedback: enableFeedback,
+    );
+  }
+
+  /// Defines the button's default appearance.
+  ///
+  /// With the exception of [ButtonStyle.side], which defines the
+  /// outline, and [ButtonStyle.padding], the returned style is the
+  /// same as for [TextButton].
+  ///
+  /// The button [child]'s [Text] and [Icon] widgets are rendered with
+  /// the [ButtonStyle]'s foreground color. The button's [InkWell] adds
+  /// the style's overlay color when the button is focused, hovered
+  /// or pressed. The button's background color becomes its [Material]
+  /// color and is transparent by default.
+  ///
+  /// All of the ButtonStyle's defaults appear below. In this list
+  /// "Theme.foo" is shorthand for `Theme.of(context).foo`. Color
+  /// scheme values like "onSurface(0.38)" are shorthand for
+  /// `onSurface.withOpacity(0.38)`. [MaterialStateProperty] valued
+  /// properties that are not followed by by a sublist have the same
+  /// value for all states, otherwise the values are as specified for
+  /// each state and "others" means all other states.
+  ///
+  /// The color of the [textStyle] is not used, the [foreground] color
+  /// is used instead.
+  ///
+  /// * `textStyle` - Theme.textTheme.button
+  /// * `backgroundColor` - transparent
+  /// * `foregroundColor`
+  ///   * disabled - Theme.colorScheme.onSurface(0.38)
+  ///   * others - Theme.colorScheme.primary
+  /// * `overlayColor`
+  ///   * hovered - Theme.colorScheme.primary(0.04)
+  ///   * focused or pressed - Theme.colorScheme.primary(0.12)
+  /// * `shadowColor` - Colors.black
+  /// * `elevation` - 0
+  /// * `padding`
```

Comment:
I think OutlinedButton padding should follow ContainedButtons (16 not 8)

Reply:
You're right, and that's what the spec says:

![Screen Shot 2020-06-26 at 11 04 20 AM](https://user-images.githubusercontent.com/1377460/85887505-cd938600-b79c-11ea-9ad3-44d4e9b1efc6.png)